Open Judge System/Web/OJS.Web/Controllers/AccountController.cs

```diff
@@ -45,30 +35,29 @@ public AccountController(IOjsData data, UserManager<UserProfile> userManager)
 
         private IAuthenticationManager AuthenticationManager => this.HttpContext.GetOwinContext().Authentication;
 
-        // GET: /Account/Login
+        [HttpGet]
```

Comment:
@jicata do you add `[HttpGet]` on get actions here?

Reply:
It seems to be a somewhat weakly followed convention (_some_ `GET` actions in _some_ controllers have the annotation, most don't). I personally don't see a reason to enforce it. What  do you think?

Reply:
In my experience, stating something explicitly is better than relying on implicitly inferring it from context, as it reduces cognitive strain on the reader.

Reply:
We skip `[HttpGet]` in Suls for sure, but I'm not sure do you have convention for this in Judge

Reply:
Even if that convention had existed, I'm inclined to forgo and abandon it, in the light that we are aiming at unifying conventions across SULS and OJS. As a result, @Teodor92 you can safely remove the annotations.